packages/shared/lib/integrations/scripts/webhook/types.ts

```diff
@@ -0,0 +1,13 @@
+import type { Config as ProviderConfig } from './../../../models/Provider.js';
```

Comment:
Split the file to avoid the circular references of "linear.ts <> webhook.manager.ts"